packages/sdk/src/conversation/notification.ts

```diff
@@ -65,6 +66,11 @@ export class Channel implements NotificationTarget {
    */
   public readonly info: ChannelInfo;
 
+  /**
+   * Team details.
+   */
+  public readonly team: TeamDetails;
```

Comment:
Why the `TeamDetails` is part of channel, not team?

Reply:
To support to get team details while finding channel, here's an example:
```ts
const channel = await teamsBot.notification.findChannel(
  c => c.info.name === "General" && c.team.name === "Contoso"
)
```

Reply:
Then how about adding team details to both Team and Channel?
It's weird if team details can be only retrieved via channel.

Reply:
Or, provide `findTeam` and `findChannel`. `findTeam` provides team details as input, while `findChannel` provides team details and channel info.

Reply:
> Then how about adding team details to both Team and Channel? It's weird if team details can be only retrieved via channel.

I noticed there's only 'Member' and 'Channel' implements the 'NotificationTarget', do we have already implemented the 'Team'

Reply:
`TeamsBotInstallation` may represent user, group, or team.

|Class|Type|Represent|
|-|-|-|
|TeamsBotInstallation|Person|1:1 chat|
|TeamsBotInstallation|Group|group chat|
|TeamsBotInstallation|Channel|team (general channel)|
|Channel|Channel|channel|
|Member|Person|1:1 chat|

My concern is - if providing team details, it should be provided for all team-related targets, including `TeamsBotInstallation(Channel)` and `Channel`.